dnf-plugin-system-upgrade.spec

```diff
@@ -43,9 +46,17 @@ This package provides the systemd services required to make the upgrade work.
 %package -n python3-%{name}
 %{?python_provide:%python_provide python3-%{name}}
 Summary:    System Upgrade plugin for DNF
-Requires:   python3-dnf >= 1.1.4
+# DNF is following Semantic Versioning, so this ensures we don't
+# get installed with a potentially incompatible version
+BuildRequires:  python3-dnf >= %{dnf_lowest_compatible}
+BuildRequires:  python3-dnf < %{dnf_not_compatible}
+BuildRequires:  python3-devel
+# TODO: change to 'python3-systemd' once that exists
+BuildRequires:  systemd-python3
```

Comment:
No, the existing subpackages would be obsoleted. It would be better to update systemd to stop producing those subpackages, but it's not strictly necessary.